module/src/common/options.ts

```diff
@@ -74,11 +73,11 @@ export function getConfig(
             }
         })
     } else {
-        settings = defaults({}, userOptions, defaultSettings)
+        settings = Object.assign({}, defaultSettings, userOptions);
```

Comment:
We could use `lodash-es` and continue using `defaults()`, no? I believe I used it because it also copies nested properties.

Reply:
I was just thinking get rid of lodash as a real dependency would be good as lodash (because it provides a lot of functionality) very often has issues which will show up here but as only defaults() is used it is not a real issue - but confusing. On the other hand I really don't see any difference between the both commands as they are here.
We only speak about object, right?
I compared them in a small test.js script: 

```
import defaults from "lodash-es/defaults.js";
const def = {
	name: 'test',
	age: 18,
	body: {
		height: 180,
		weight: 70
	},
	hobby: ['basketball', 'football'],
	devices: {
		phone: {
			brand: 'iphone',
			model: 'iphone 11'
		},
		computer: {
			brand: 'macbook',
			model: 'macbook pro 2019'
		}
	}
};
const override = {
	hobby: ['gaming'],
	body: {
		height: 190
	}, devices: {
		phone: {
			model: 'iphone 10 SE'
		}
	}
}

console.log("default object def:");
console.dir(def, { depth: 5 });
console.log("object that overwrite defaults:");
console.dir(override, { depth: 5 });
console.log("Object.assign:");
console.dir(Object.assign({}, def, override), { depth: 5 });
console.log("defaults:");
console.dir(defaults({}, override, def), { depth: 5 });
```
The result is: 
```
default object def:
{
  name: 'test',
  age: 18,
  body: { height: 180, weight: 70 },
  hobby: [ 'basketball', 'football' ],
  devices: {
    phone: { brand: 'iphone', model: 'iphone 11' },
    computer: { brand: 'macbook', model: 'macbook pro 2019' }
  }
}
object that overwrite defaults:
{
  hobby: [ 'gaming' ],
  body: { height: 190 },
  devices: { phone: { model: 'iphone 10 SE' } }
}
Object.assign:
{
  name: 'test',
  age: 18,
  body: { height: 190 },
  hobby: [ 'gaming' ],
  devices: { phone: { model: 'iphone 10 SE' } }
}
defaults:
{
  hobby: [ 'gaming' ],
  body: { height: 190 },
  devices: { phone: { model: 'iphone 10 SE' } },
  name: 'test',
  age: 18
}
```

If a more deeper merge is needed it could be done like that:
```
import defaults from "lodash-es/defaults.js";
const def = {
	name: 'test',
	age: 18,
	body: {
		height: 180,
		weight: 70
	},
	hobby: ['basketball', 'football'],
	devices: {
		phone: {
			brand: 'iphone',
			model: 'iphone 11'
		},
		computer: {
			brand: 'macbook',
			model: 'macbook pro 2019'
		}
	}
};
const override = {
	hobby: ['gaming'],
	body: {
		height: 190
	}, devices: {
		phone: {
			model: 'iphone 10 SE'
		}
	}
}

function isObject(item) {
	return (item && typeof item === 'object' && !Array.isArray(item));
}

function mergeDeep(target, ...sources) {
	if (!sources.length) {
		return target;
	}
	const source = sources.shift();

	if (isObject(target) && isObject(source)) {
		for (const key in source) {
		if (isObject(source[key])) {
			if (!target[key]) Object.assign(target, { [key]: {} });
			mergeDeep(target[key], source[key]);
		} else {
			Object.assign(target, { [key]: source[key] });
		}
		}
	}
	return mergeDeep(target, ...sources);
}

console.log("mergeDeep:");
console.dir(mergeDeep({}, def, override), { depth: 5 });
```
With that deeper merge the result would be:
```
default object def:
{
  name: 'test',
  age: 18,
  body: { height: 180, weight: 70 },
  hobby: [ 'basketball', 'football' ],
  devices: {
    phone: { brand: 'iphone', model: 'iphone 11' },
    computer: { brand: 'macbook', model: 'macbook pro 2019' }
  }
}
object that overwrite defaults:
{
  hobby: [ 'gaming' ],
  body: { height: 190 },
  devices: { phone: { model: 'iphone 10 SE' } }
}
mergeDeep:
{
  name: 'test',
  age: 18,
  body: { height: 190, weight: 70 },
  hobby: [ 'gaming' ],
  devices: {
    phone: { brand: 'iphone', model: 'iphone 10 SE' },
    computer: { brand: 'macbook', model: 'macbook pro 2019' }
  }
}
```
What you think about this?